packages/babel-plugin-proposal-optional-chaining/src/index.js

```diff
@@ -1,4 +1,7 @@
 import { declare } from "@babel/helper-plugin-utils";
+import skipTransparentExprWrappers, {
+  getCallContext,
+} from "@babel/helper-skip-transparent-expr-wrappers";
```

Comment:
Nit: I would slightly prefer to only use named exports in `@babel/helper-skip-transparent-expr-wrappers`, to have nicer imports.

Reply:
Sorted!